eslint.config.mjs

```diff
@@ -38,7 +38,7 @@ const config = createConfig([
       'apps/storybook-react/.storybook/*.tsx',
```

Comment:
Updating in eslint config file. Also checked for any remaining tailwind.config.js instances within the monorepo. There are a couple in docs but I think we should keep them as it's the more general way of using tailwind

<img width="440" height="339" alt="Screenshot 2025-07-14 at 3 29 43 PM" src="https://github.com/user-attachments/assets/b5a79dea-2e10-463c-b7da-f618e8aa6f34" />